cmd/osm-controller/certificates_test.go

```diff
@@ -109,6 +140,16 @@ var _ = Describe("Test CMD tools", func() {
 			ns := uuid.New().String()
 			secretName := uuid.New().String()
 
+			certProviderUtil := &certproviderutil.CertProviderUtil{
+				KubeClient:            kubeClient,
+				CertProviderKind:      osmCertificateManagerKind,
+				CertProviderNamespace: ns,
+				CABundleSecretName:    secretName,
+
+				VaultOptions:       vaultOptions,
+				CertManagerOptions: certManagerOptions,
+			}
+
```

Comment:
We could potentially have a method that does the initialization and invoke that in all the tests

Reply:
Yes I thought about this. I can update this in a follow-up since I have more changes that need to go into this pkg. I didn't create the initialization function because it might not be necessary to specify all options together, so we would end up having to pass empty structs for certain fields.